packages/bits/package.json

```diff
@@ -138,12 +138,12 @@
     "@angular/platform-browser-dynamic": "^11.1.2",
     "@angular/router": "^11.1.2",
     "@applitools/eyes-protractor": "1.9.1",
+    "@axe-core/webdriverjs": "4.1.1",
```

Comment:
should we add the `^` to the beginning so that we can automatically receive minor updates like security fixes?

Reply:
good idea, will add in a sec